sdk/attestation/attestation/src/utils/base64.browser.ts

```diff
@@ -84,10 +84,34 @@ export function hexToByteArray(value: string): Uint8Array {
   return Uint8Array.from(byteArray);
 }
 
+export function byteArrayToHex(value: Uint8Array): string {
+  return value.reduce((str, byte) => str + byte.toString(16).padStart(2, "0"), "");
+}
+
 /**
  * Converts a hex encoded string to its base64 equivalent.
  * @param value - Hex encoded value
  */
 export function hexToBase64(value: string): string {
   return base64EncodeByteArray(hexToByteArray(value));
 }
+
+export enum PemType {
+  Certificate = "CERTIFICATE",
+  PrivateKey = "PRIVATE KEY"
+}
```

Comment:
This could be just a union of strings:
```suggestion
export PemType = "CERTIFICATE" | "PRIVATE KEY";
```

Reply:
Please consider doing this to all enums.

Reply:
I'm not sure that losing type safety is worth the change, but whatever.

Reply:
This type specifies the same thing as the original enum, there is no loss of type safety here.